File: PetClinicApp/PetClinic.Domain/Entities/Pet.cs
namespace PetClinic.Domain.Entities
{
    using PetClinic.Domain.AbstracClasses;

    public class Pet : Animal
    {
        public int Id { get; set; }

        public string Name { get; set; } = null!;

        public int Age { get; set; }

        public int OwnerId { get; set; }
        public Owner Owner { get; set; } = null!;

        public int? DoctorId { get; set; }
        public Doctor? Doctor { get; set; }

        public int? ReceptionistId { get; set; }
        public Receptionist? Receptionist { get; set; }

        public bool IsDeleted { get; set; } = false;
    }
}

File: PetCareApp/PetCare-FE/src/components/Pets/Pet.js
function Pet({
    pet
}) {
    return (
        <>
            <tr>
                <td>{pet.name}</td>
                <td>{pet.age}</td>
                <td>{pet.ownerFullName}</td>
                <td>{pet.type} </td>
                <td>{pet.breed}</td>
            </tr>
        </>
    )
}

export default Pet;